CustomerOrganizer/CustomerOrganizer/CustomerClass.py

```diff
@@ -99,7 +99,10 @@ def save(self, new_data, new_vehicle, new_notes):
             return
         
         try:
-            self.file.save_file(total_data, customer_last_name + "_" + customer_first_name[0] + self.vehicle["VIN"][0:2])
+            if not save_as:
+                self.file.save_file(total_data, customer_last_name + "_" + customer_first_name[0] + self.vehicle["VIN"][0:2])
+            else:
+                self.file.save_as(total_data, window)
```

Comment:
Running into the error pop-up bug when pressing cancel without saving the file. We can put a try-block here to prevent it from showing up, like this:

`try:
      self.file.save_as(total_data, window)
 except:
      return`

That's a try-block in a try-block though. I'm not sure if that's considered messy or an okay convention.